src/AzureExtension/AzureExtension.csproj

```diff
@@ -68,6 +68,7 @@
     <PackageReference Include="Microsoft.Data.Sqlite" Version="7.0.4" />
     <PackageReference Include="Microsoft.Identity.Client" Version="4.56.0" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="4.56.0" />
+    <PackageReference Include="Microsoft.ML.Tokenizers" Version="0.22.0-preview.24271.1" />

```

Comment:
@krschau / @EricJohnson327, ah, sorry, looks like the Tokenizer library also depends on a Google protocol buffer library too, which is not found on the feed. Would you mind adding that package as well? 

![image](https://github.com/microsoft/DevHomeAzureExtension/assets/8368026/de1fd236-02e2-4c5f-bb5f-45d4ce72be28)

The logs have some complaints about System.Text.Json as well, which I hope are red herrings since I see version 8.0.0 of that package on the feed.
https://github.com/microsoft/DevHomeAzureExtension/actions/runs/9407634939/job/25916308719?pr=205

Reply:
Thanks for adding the protocol buffer library. I saw that the CI build failed, due to its unfulfilled demand for 6.0.1 of System.Text.Json. The feed already has version 8.0.0 on it but that is apparently not sufficient (I don't know why as yet). I tried re-running with more diagnostics, but that didn't yield further insight. https://github.com/microsoft/DevHomeAzureExtension/actions/runs/9407634939/job/25951443891

I don't see any transitive dependencies of this library that would strictly need 6.0.1 (Google.Protobuf doesn't itself depend on System.Text.Json so as best as I can tell, the dependency is being introduced by the Tokenizers library... but Visual Studio's package explorer says it's a >= relationship, which I interpreted to mean that higher versions are also ok). 

Do either of you all have any ideas? I'm not super Nuget-savvy so I'm stumped on what to look for next. 

https://github.com/microsoft/DevHomeAzureExtension/actions/runs/9407634939/job/25945816955?pr=205

Reply:
We've had to add lower versions in the past for stuff like this. We can do that, unless @EricJohnson327 knows some nuget magic.

Reply:
Thanks! @EricJohnson327, do you have any ideas here? If not, could either you or @krschau try adding the lower version of the JSON library to the feed? (I'm curious if that will cause problems with other parts of the extension either at build or runtime, e.g., if they are relying on functionality only found in the higher version for example...)

Reply:
It is possible we just need to add an explicit reference to System.Text.Json here in our csproj to make it use the version we want rather than the minimum needed by our dependency.

Reply:
Thanks! Let me try making the reference explicit in the .csproj and let's see if that gets the build unblocked.

Reply:
@manodasanW was spot-on with that recommendation! I didn't realize that NuGet would default to the lowest-supported version (I was thinking that it might also take into consideration what is published / available on the feed as part of making that decision); lesson learned. :)